### DISCUSSION TITLE
Dynamic hulls!

### DISCUSSION THREAD

**Discussion by Psyduckdeluxe** [FakeFishGames/Barotrauma#10345] [Ideas and suggestions]
Since i noticed and attempting to exploit that docking ports creates a hull when docked.

And the dry dock is every sub builders wet dream. And to me it feels like some parts of it is already in the game!

I was wondering if we could have dynamic hulls that could be turned on and off like the ones built in docking ports but without having to jump as many problems  to disable or enable a hull!

**Comment by iridescent-lightning** [comment]
Interesting. I never knew that the docking can generate a new hull
